Source/WebCore/contentextensions/ContentExtensionsBackend.cpp

```diff
@@ -256,23 +245,11 @@ ContentRuleListResults ContentExtensionsBackend::processContentRuleListsForLoad(
     URL frameURL;
     bool mainFrameContext = false;
     RequestMethod requestMethod = readRequestMethod(initiatingDocumentLoader.request().httpMethod()).value_or(RequestMethod::None);
-    Vector<URL> ancestorSubframeURLs;
 
-    if (RefPtr frame = initiatingDocumentLoader.frame()) {
+    if (auto* frame = initiatingDocumentLoader.frame()) {
```

Comment:
This could stay, but since you're doing a pure revert, I think it's fine to revert this.